package.json

```diff
@@ -2,22 +2,22 @@
   "name": "styled-system",
   "version": "4.1.1",
   "description": "Responsive, theme-based style props for building design systems with React",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
+  "main": "dist/styled-system.cjs.js",
+  "module": "dist/styled-system.es.js",
```

Comment:
Please keep `esm` prefix. It's more descriptive. `es` is often used to distribute es6 syntax.